lang/java/reef-bridge-client/src/main/java/org/apache/reef/bridge/client/DriverStatusHTTPHandler.java

```diff
@@ -52,8 +55,35 @@
    */
   private ReefServiceProtos.JobStatusProto lastStatus = null;
 
+  /**
+   * The clock is used to schedule a check whether the handler has been called.
+   */
+  private final Clock clock;
+
+  /**
+   * The maximum number of times the AlarmHandler will be scheduled.
+   */
+  private static final int MAX_RETRIES = 10;
+
+  /**
+   * The interval between alarms.
+   */
+  private static final int ALARM_INTERVAL = 200; //ms
```

Comment:
Same as in C#: maybe `MAX_RETRIES` and `ALARM_INTERVAL` should be injectable parameters with `10` and `200 // ms` as default values?

Reply:
Done.